### PR TITLE
Added Dockerfile to expand database

### LINKED ISSUE #37: Create Dockerfile
**Is your feature request related to a problem? Please describe.**
Dockerfile is needed to expand PostgreSQL database

**Describe the solution you'd like**
* Dockerfile
* ReadMe how to install

### PR DESCRIPTION
**Pull request by MykytaKyt**
# What does this PR do?

Dockerfile is needed to expand PostgreSQL database.

## Issue reference

Fixes #37

## Explanation

To use PostgreSQL with docker

### CODE REVIEW

File: datakyt/db/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# PostgreSQL docker image with a database for a Datakyt project
+
+This image contains an initial database schema.
+
+## Build image
+
+To build this image simply run `docker build -t datakyt/docker-postgres:<version> .`
```
Comment (aartiukh) [inline datakyt/db/README.md:7]: What's `version`? Please elaborate.

Comment (MykytaKyt) [reply datakyt/db/README.md:7]: I mean the PostgreSQL version.  In our case we are using version 12.

Comment (aartiukh) [reply datakyt/db/README.md:7]: @MykytaKyt From docker [docs](https://docs.docker.com/engine/reference/commandline/build/):
 > --tag , -t  Name and optionally a tag in the ‘name:tag’ format

So this shouldn't be a PostgreSQL version, this tag is used for specifying of different docker image versions that may correspond to the git tags, for example. Say you have DataKyt 1.0 and 2.0 versions, that you can create docker images with different tags.

**Review by aartiukh — APPROVED** [review]

**Review by OleksandrZhytnyk — APPROVED** [review]

**Review by paulzcooper — APPROVED** [review]

**Comment by aartiukh** [comment]
<details>
  <summary>Build</summary>
  
  ```
  aa@aa-pc:~/Downloads/datakyt/datakyt/db$ sudo docker build -t datakyt/docker-postgres .
  [sudo] password for aa:
  Sending build context to Docker daemon  14.34kB
  Step 1/3 : FROM postgres:12
  12: Pulling from library/postgres
  a076a628af6f: Pull complete
  d54fa0e0eb76: Pull complete
  8950a7ea6876: Pull complete
  1cc02b1df09e: Pull complete
  06f548e51228: Pull complete
  74665692d4ea: Pull complete
  235e34eb13ab: Pull complete
  4ad7ef5e2e79: Pull complete
  6beb6e5906f0: Pull complete
  ceef368dc969: Pull complete
  2b9de13efdd7: Pull complete
  f65d6bb87010: Pull complete
  4a3c197c71df: Pull complete
  f163c13f9df6: Pull complete
  Digest: sha256:f9af02fa377c201021a66d7e0cc087af437604b053f6517dc018d48a75e39e5e
  Status: Downloaded newer image for postgres:12
   ---> 9fe1bb170ece
  Step 2/3 : COPY create_database.sql /docker-entrypoint-initdb.d
   ---> 7f8977599061
  Step 3/3 : COPY create_table.sql /docker-entrypoint-initdb.d
   ---> 1b840db06cce
  Successfully built 1b840db06cce
  Successfully tagged datakyt/docker-postgres:latest
  ```
</details>

<details>
  <summary>Run</summary>

  ```
  aa@aa-pc:~/Downloads/datakyt/datakyt/db$ sudo docker run -e POSTGRES_PASSWORD="12345" datakyt/docker-postgres
  The files belonging to this database system will be owned by user "postgres".
  This user must also own the server process.
  
  The database cluster will be initialized with locale "en_US.utf8".
  The default database encoding has accordingly been set to "UTF8".
  The default text search configuration will be set to "english".
  
  Data page checksums are disabled.
  
  fixing permissions on existing directory /var/lib/postgresql/data ... ok
  creating subdirectories ... ok
  selecting dynamic shared memory implementation ... posix
  selecting default max_connections ... 100
  selecting default shared_buffers ... 128MB
  selecting default time zone ... Etc/UTC
  creating configuration files ... ok
  running bootstrap script ... ok
  performing post-bootstrap initialization ... ok
  syncing data to disk ... ok
  initdb: warning: enabling "trust" authentication for local connections
  
  You can change this by editing pg_hba.conf or using the option -A, or
  --auth-local and --auth-host, the next time you run initdb.
  
  Success. You can now start the database server using:
  
      pg_ctl -D /var/lib/postgresql/data -l logfile start
  
  waiting for server to start....2021-01-21 06:42:45.748 UTC [47] LOG:  starting PostgreSQL 12.5 (Debian 12.5-1.pgdg100+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 8.3.0-6) 8.3.0, 64-bit
  2021-01-21 06:42:45.754 UTC [47] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
  2021-01-21 06:42:45.791 UTC [48] LOG:  database system was shut down at 2021-01-21 06:42:44 UTC
  2021-01-21 06:42:45.801 UTC [47] LOG:  database system is ready to accept connections
   done
  server started
  
  /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/create_database.sql
  CREATE DATABASE
  CREATE ROLE
  GRANT
  
  
  /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/create_table.sql
  SET
  GRANT
  CREATE TABLE
  CREATE TABLE
  CREATE TABLE
  CREATE TABLE
  CREATE TABLE
  CREATE TABLE
  CREATE TABLE
  CREATE TABLE
  CREATE TABLE
  CREATE TABLE
  CREATE TABLE
  CREATE TABLE
  CREATE TABLE
  
  
  2021-01-21 06:42:46.855 UTC [47] LOG:  received fast shutdown request
  waiting for server to shut down....2021-01-21 06:42:46.862 UTC [47] LOG:  aborting any active transactions
  2021-01-21 06:42:46.864 UTC [47] LOG:  background worker "logical replication launcher" (PID 54) exited with exit code 1
  2021-01-21 06:42:46.866 UTC [49] LOG:  shutting down
  2021-01-21 06:42:47.062 UTC [47] LOG:  database system is shut down
   done
  server stopped
  
  PostgreSQL init process complete; ready for start up.
  
  2021-01-21 06:42:47.203 UTC [1] LOG:  starting PostgreSQL 12.5 (Debian 12.5-1.pgdg100+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 8.3.0-6) 8.3.0, 64-bit
  2021-01-21 06:42:47.204 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
  2021-01-21 06:42:47.204 UTC [1] LOG:  listening on IPv6 address "::", port 5432
  2021-01-21 06:42:47.216 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
  2021-01-21 06:42:47.245 UTC [84] LOG:  database system was shut down at 2021-01-21 06:42:47 UTC
  2021-01-21 06:42:47.255 UTC [1] LOG:  database system is ready to accept connections
  ```

</details>

@MykytaKyt good job! Please go ahead and merge this PR.

**Pull request merged**